datasets/e_commerce/e_commerce.py

```diff
@@ -0,0 +1,70 @@
+import csv
+import os
+
+import datalabs
+from datalabs import get_task, TaskType
+
+
+_CITATION = """\
+@inproceedings{zhang2018dua,
+    title = {Modeling Multi-turn Conversation with Deep Utterance Aggregation},
+    author = {Zhang, Zhuosheng and Li, Jiangtong and Zhu, Pengfei and Zhao, Hai},
+    booktitle = {Proceedings of the 27th International Conference on Computational Linguistics (COLING 2018)},
+    pages={3740--3752},
+    year = {2018}
+}
+"""
+
+_DESCRIPTION = """\
+E-commerce Dialogue Corpus, comprising a training data set, a development set and a test set for retrieval based chatbot.
+"""
+_TRAIN_DOWNLOAD_URL="https://cdatalab1.oss-cn-beijing.aliyuncs.com/dialogue/E-commerce/train.txt"
+_VALIDATION_DOWNLOAD_URL="https://cdatalab1.oss-cn-beijing.aliyuncs.com/dialogue/E-commerce/dev.txt"
+_TEST_DOWNLOAD_URL="https://cdatalab1.oss-cn-beijing.aliyuncs.com/dialogue/E-commerce/test.txt"
+
+
+class E_commerce(datalabs.GeneratorBasedBuilder):
+   
+    VERSION = datalabs.Version("1.0.0")
+
+    def _info(self):
+        
+        return datalabs.DatasetInfo(
+            description=_DESCRIPTION,
+            features=datalabs.Features(
+                {
+                "Context": datalabs.features.Sequence(datalabs.Value("string")),
+                "Utterance": datalabs.Value("string"),
+                "Label": datalabs.Value("int32") 
+                }
+            ),
+            supervised_keys=None,
```

Comment:
* it seems that we didn't define task_template for this task
* also it would be nice if feature named could be fully-lowercased.